variables.tf

```
@@ -320,3 +320,9 @@ variable "wait_for_deployment" {
   default     = true
   description = "When set to 'true' the resource will wait for the distribution status to change from InProgress to Deployed"
 }
+
+variable "enable_encryption" {
```

```suggestion
variable "encryption_enabled" {
```

we usually use terms like `xxx_enabled` in this case